frontend/javascripts/oxalis/view/action-bar/toolbar_view.tsx

```diff
@@ -582,6 +620,13 @@ function ChangeBrushSizeButton() {
     );
   };
 
+  const extendedKeyboardBindings = {
+    1: () => handleUpdateBrushSize(smallBrushSize),
+    2: () => handleUpdateBrushSize(mediumBrushSize),
+    3: () => handleUpdateBrushSize(largeBrushSize),
+  };
+  new InputKeyboardNoLoop({}, {}, extendedKeyboardBindings);
```

Comment:
I'd move the shortcut related code into the plane_controller.tsx module. In `getNotLoopedKeyboardControls`, `extendedControls` are defined where you can add the three shortcuts. The handle-code needs to be redefined there probably, as you need to read out the brush sizes from the store.
Now the explanation as to why this should be moved: Instantiating a new class without really using the object afterwards is usually a code smell. In this case, the instance will linger around and will never be cleaned up again (via the `destroy` method which the object offers). Even worse, on each render of this component, a new InputKeyboardNoLoop instance will be created (and all will be listening to the shortcuts).